src/app/components/category-products/category-products.component.scss

```diff
@@ -0,0 +1,3 @@
+.dnone {
+    display: none !important;
```

Comment:
there is aleady utility class for it i think it is called d-none

Reply:
I can't use "-" in ngClass

Reply:
of course you can